src/components/common/slider-range/slider-range.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
```

Comment:
I am afraid, the range slider cannot work with indexes 0..max if we accept unequal intervals. It will probably have to handle dates by itself.

Reply:
Resolved => Slider range has now its own state, it now handles percentage values.